### DISCUSSION TITLE
Why can't the remote computer transfer files to the local machine

### DISCUSSION THREAD

**Discussion by zukxu** [rustdesk/rustdesk#1507] [Q&A]
![image](https://user-images.githubusercontent.com/63381545/189863724-2a2d2494-f6d1-4442-9f56-ffb06aec04fc.png)

when the remote computer transfer files to the local machine ,it's always waiting

**Comment by rustdesk** [comment]
@Heap-Hop both Windows should not have https://github.com/rustdesk/rustdesk/issues/1080 issue.

**Comment by Heap-Hop** [comment]
@zukxu  What are the rustdesk versions of the two devices?

**Reply by zukxu** [reply]
The rustdesk version of both machines is 1.1.9

![image](https://user-images.githubusercontent.com/63381545/190208847-b4a3fa9f-7446-4386-9566-54b26fc9d0c3.png)

**Reply by coybersuh** [reply]
having the same problem, same version

**Reply by rustdesk** [reply]
try out nightly build please

**Reply by coybersuh** [reply]
> try out nightly build please

will try it out, thanks for the feedback

**Reply by Poikilos** [reply]
> > try out nightly build please
> 
> will try it out, thanks for the feedback

Did the nightly version work? I'm having trouble with 1.1.9 but can't drive over and put nightly on the other computer today and want to know if its worth installing a non-release version at a business.
